salt/returners/local_cache.py

```diff
@@ -100,12 +100,13 @@ def prep_jid(nocache=False, passed_jid=None, recurse_count=0):
 
     # Make sure we create the jid dir, otherwise someone else is using it,
     # meaning we need a new jid.
-    try:
-        os.makedirs(jid_dir_)
-    except OSError:
```

Comment:
It would be good to filter for the specific ``OSError`` of ``errno.EEXIST`` where the ``JID`` dir is already created because it is already in-use by another job.  All other ``OSError``s (how about ``errno.EPERM``) should be raised as a real ``SaltCacheError``.